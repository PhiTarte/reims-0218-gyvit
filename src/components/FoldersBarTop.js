import React from "react"
import {
  Navbar,
  Nav,
  NavbarBrand,
  NavbarToggler,
  Collapse,
  NavItem,
  Button
} from "reactstrap"

const FoldersBarTop = () => {
  return (
    <Navbar
      expand="xs"
      style={{
        "margin-top": "54px"
      }}
    >
      <NavbarBrand
        style={{
          width: "45px",
          height: "24px",
          "margin-left": "215px"
        }}
      >
        Folders
      </NavbarBrand>
      <NavbarToggler onClick={this.toggle} />
      <Collapse navbar>
        <Nav className="ml-auto" navbar>
          <NavItem>
            <Button
              style={{
                width: "50px",
                height: "50px",
                "margin-right": "235px",
                "border-radius": "50%",
                "background-color": "#725fe3"
              }}
            >
              +
            </Button>
          </NavItem>
        </Nav>
      </Collapse>
    </Navbar>
  )
}
export default FoldersBarTop
