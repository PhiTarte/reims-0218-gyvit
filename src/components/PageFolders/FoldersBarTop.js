import React from "react"
import {
  Col,
  Navbar,
  Nav,
  NavbarBrand,
  NavbarToggler,
  Collapse,
  NavItem,
  Button,
  Row,
  Container
} from "reactstrap"
import ModalCreateDirContainer from "../../containers/ModalCreateDirContainer"
import "./styles-page-folders/folderBarTop.css"

const FoldersBarTop = () => {
  return (
    <Container>
      <ModalCreateDirContainer />
      <Row>
        <Col xs="12">
          <Navbar
            expand="xs"
            style={{
              marginTop: "54px"
            }}
          >
            <NavbarBrand>Folders</NavbarBrand>
            <NavbarToggler />
            <Collapse navbar>
              <Nav className="ml-auto" navbar>
                <NavItem>
                  <Button className="ButtonFoldersBarTop">+</Button>
                </NavItem>
              </Nav>
            </Collapse>
          </Navbar>
        </Col>
      </Row>
    </Container>
  )
}
export default FoldersBarTop
