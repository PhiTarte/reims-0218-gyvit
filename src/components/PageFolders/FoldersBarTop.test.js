import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import FoldersBarTop from "./FoldersBarTop"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<FoldersBarTop />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<FoldersBarTop />).toJSON()
  expect(tree).toMatchSnapshot()
})
