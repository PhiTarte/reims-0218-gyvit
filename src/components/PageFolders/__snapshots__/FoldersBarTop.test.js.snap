// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  className="row"
>
  <div
    className="col-12"
  >
    <nav
      className="navbar navbar-expand"
      style={
        Object {
          "margin-top": "54px",
        }
      }
    >
      <a
        className="navbar-brand"
      >
        Folders
      </a>
      <button
        className="navbar-toggler"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
      <div
        className="collapse navbar-collapse"
        style={Object {}}
      >
        <ul
          className="ml-auto navbar-nav"
        >
          <li
            className="nav-item"
          >
            <button
              className="btn btn-secondary"
              onClick={[Function]}
              style={
                Object {
                  "background-color": "#725fe3",
                  "border-radius": "50%",
                }
              }
              type={undefined}
            >
              +
            </button>
          </li>
        </ul>
      </div>
    </nav>
  </div>
</div>
`;
